pom.xml

```diff
@@ -175,4 +190,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-</project>  
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <configuration>
+                    <compatibleSinceVersion>2.0</compatibleSinceVersion>
```

Comment:
Really... that is the stated purpose of the attribute and I use it every time there is a field migration... adding new fields is not a warning